protolib/traverse.py

```diff
@@ -81,7 +110,12 @@ def get_protocol_pyfile(protocol: dict):
     """Returns path to python entry script for a protocl dir.
     Ignores python files that start with "test_"
     """
-    pyfiles = protocol['detected-files']['protocol']
+    pyfiles = protocol['detected-files'].get('OT 1 protocol')
```

Comment:
Not sure how this passed, but file handle name is different than what we're checking the pyfile on ('OT_1_PROTOCOL' vs OT 1 protocol